node/src/service.rs

```diff
@@ -123,6 +141,20 @@ pub fn new_full(mut config: Configuration) -> Result<TaskManager, ServiceError>
 		};
 	}
 
+	//dht
+	let local_key = identity::Keypair::generate_ed25519();
+    let local_peer_id = PeerId::from_public_key(local_key.public().clone());
+
+	//address
+    let localhost = Ipv4Addr::new(0, 0, 0, 0);
+    let test_tcp: &str = "8080";
+    let targetted_tcp = Protocol::Ip4(localhost);
+    
+    let address = format!("{}/tcp/{}/p2p/{}", targetted_tcp, test_tcp, local_peer_id.clone());
+    let multi_addr= address.as_str();
+
+	let (peer_id, multiaddr) = config::parse_str_addr(multi_addr).unwrap();
```

Comment:
Where we have used this peer_id?
I think it will generate the new one rather than generated from the network below.

Reply:
that would be removed. I was initially using it while working on the code. We would be using default node peerId instead